### PR TITLE
Adds More Animations

### PR DESCRIPTION
**Pull request by SDTheCyanWyan**
**When merged this pull request will:**

- Add new animations from vanilla ArmA 3 to be used in Ambient Animations module.
- Capitalised and beautified name of animations.
- Removed `"KNEEL"` as it is unused (?).
- Added new animations.

### CODE REVIEW

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_ambientAnimGetParams.sqf [hunk 1]
```diff
@@ -21,10 +21,12 @@ private _anim_set = _this;
 private _return = switch (_anim_set) do
 {   // [anims, noWeapon]
 	case "STAND_1":	{ [["HubStanding_idle1","HubStanding_idle2","HubStanding_idle3"], false] };
-	case "STAND_2": { [["amovpercmstpslowwrfldnon","amovpercmstpslowwrfldnon","aidlpercmstpslowwrfldnon_g01","aidlpercmstpslowwrfldnon_g02","aidlpercmstpslowwrfldnon_g03","aidlpercmstpslowwrfldnon_g05"], false] };
+	case "STAND_2": { [["amovpercmstpslowwrfldnon","amovpercmstpslowwrfldnon","aidlpercmstpslowwrfldnon_g01","aidlpercmstpslowwrfldnon_g02","aidlpercmstpslowwrfldnon_g03","aidlpercmstpslowwrfldnon_g05","Acts_AidlPercMstpSloWWrflDnon_warmup_3_loop"], false] };
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_ambientAnimGetParams.sqf]:
```suggestion
	case "STAND_2": { [["amovpercmstpslowwrfldnon","amovpercmstpslowwrfldnon","aidlpercmstpslowwrfldnon_g01","aidlpercmstpslowwrfldnon_g02","aidlpercmstpslowwrfldnon_g03","aidlpercmstpslowwrfldnon_g05","Acts_AidlPercMstpSloWWrflDnon_warmup_3_loop"], false] };
```

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf [hunk 2]
```diff
@@ -24,36 +24,47 @@ private _dialogResult =
...
+				"Dead (Erect)",
+				"Weapon Ready (Kneel)",
+				"Leaned On Table",
+				"Bored",
+				"Sit (Binoc)",
+				"Sit (Weapon)",
+				"Squat (Weapon)",
+				"Squat",
+				"Guard (Pistol)"
+
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf]: Remove line

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf [hunk 3]
```diff
@@ -65,7 +76,11 @@ private _persistent = _dialogResult select 1;
 _anim = switch (_dialogResult select 0) do
 {
 	case 0:	{"TERMINATE"};
-	case 1: {"SIT_LOW"};
+	case 1: 
+	{
+		selectRandom ["SIT_LOW_1", "SIT_LOW_2", "SIT_LOW_3", "SIT_LOW_4", "SIT_LOW_5", "SIT_LOW_6"]; 
+
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf]: Remove new line

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf [hunk 4]
```diff
@@ -111,6 +126,30 @@ _anim = switch (_dialogResult select 0) do
...
+		selectRandom ["SIT_WEAP_1", "SIT_WEAP_2", "SIT_WEAP_3", "SIT_WEAP_4"];
+	};
+	case 36: {"SQUAT_WEAP"};
+	case 37: {"SQUAT"};
+	case 38: 
+	{
+		selectRandom ["STAND_GUARD_P1", "STAND_GUARD_P2", "STAND_GUARD_P3", "STAND_GUARD_P4"];
+	};
+
+
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf]: Remove new lines

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf [hunk 5]
```diff
@@ -111,8 +124,29 @@ _anim = switch (_dialogResult select 0) do
...
+		selectRandom ["SIT_WEAP_1", "SIT_WEAP_2", "SIT_WEAP_3", "SIT_WEAP_4"];
+	};
+	case 36: {"SQUAT_WEAP"};
+	case 37: {"SQUAT"};
+	case 38: 
+	{
+		selectRandom ["STAND_GUARD_P1", "STAND_GUARD_P2", "STAND_GUARD_P3", "STAND_GUARD_P4"];
+	};
 };
-
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_animation.sqf]: Don't want to be pedantic, but you could have left one newline in. As the original source.

**Comment by CreepPork** [comment]
And is it tested?

**Comment by SDTheCyanWyan** [comment]
Yeah tested in both. Kneel is not used anywhere, the identifier is not even referenced to in animation.sqf

**Comment by SDTheCyanWyan** [comment]
Neither are the animations in kneel used

**Comment by CreepPork** [comment]
@neilzar @oOKexOo

**Pull request merged**